packages/core/src/Stores/ui-store.js

```diff
@@ -6,6 +6,7 @@ import BaseStore from './base-store';
 const store_name = 'ui_store';
 
 export default class UIStore extends BaseStore {
+    url_hashed_values = '';
```

Comment:
@rupato-deriv Why do we need to keep it in the store? 🤔

Reply:
@farzin-deriv because this is the inly place where we can access the hash value because there is a race condition else it gets cleared.